packages/stryker-karma-runner/src/KarmaConfigReader.ts

```diff
@@ -34,10 +34,10 @@ export default class KarmaConfigReader {
     this.parseNativeKarmaConfig();
   }
 
-  private readConfig(): karma.ConfigOptions {
+  private readConfig(karmaConfigFile: string): karma.ConfigOptions {
```

Comment:
Why do you pass this as a parameter?

Reply:
Because of strict null checks, my friend